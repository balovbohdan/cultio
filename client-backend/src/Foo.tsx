import * as React from 'react';

export const Foo = ({name}) => <h1>{name}</h1>;