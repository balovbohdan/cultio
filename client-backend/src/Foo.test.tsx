import * as React from 'react';
import { shallow } from 'enzyme';

import {Foo} from './Foo';

// @ts-ignore
describe('Foo', () => {
    // @ts-ignore
    it('test Foo', () => {
        const component = shallow(<Foo name={'name'}/>);

        // @ts-ignore
        expect(component.text()).toEqual('name');
    })
});