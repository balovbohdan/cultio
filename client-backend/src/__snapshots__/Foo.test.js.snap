// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo Should render correctly. 1`] = `
<h1>
  Foo, 
  Bohdan
  !
</h1>
`;

exports[`Foo should render correctly in "debug" mode 1`] = `
<h1>
  Foo, 
  name
  !
</h1>
`;
